pkg/apiserver/rest/usecase/config.go

```diff
@@ -278,6 +281,26 @@ func SyncConfigs(ctx context.Context, k8sClient client.Client, project string, t
 			return err
 		}
 		// config sync application doesn't exist, create one
+		clusterTargets := convertClusterTargets(targets)
+		if len(clusterTargets) == 0 {
+			errMsg := "no policy (no targets found) to sync configs"
+			klog.InfoS(errMsg, "project", project)
+			return errors.New(errMsg)
```

Comment:
If no targets, should return nil?

Reply:
Without targets, the application won't have any policies, the application will fail which means the sync of config will fail. Users have to redeploy the application to make it work.

Reply:
I mean without targets there is no need for configuration synchronization and not block deploy the app.